chain/client/src/client_actor.rs

```diff
@@ -87,7 +87,7 @@ use rand::{Rng, thread_rng};
 use std::fmt;
 use std::sync::Arc;
 use tokio::sync::broadcast;
-use tracing::{debug, debug_span, error, info, trace, warn};
+use tracing::debug_span;
```

Comment:
same for the spans, should we also use `tracing::debug_span` inline instead of importing?